plugin.py

```
@@ -2007,260 +2036,229 @@ def celery_generate_all():
+
+                if _is_vod_refresh_complete(acct):
+                    LOGGER.info(
+                        "Auto-monitor: VOD refresh detected for account '%s' (id=%s, updated=%s, msg='%s')",
+                        acct.name, acct.id, updated, (acct.last_message or "")[:80]
+                    )
+                    triggered = True
+
+        if triggered:
+            _auto_run_generation()
+
+    except Exception as e:
```

Ruff (0.14.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)